vault/activity_log.go

```diff
@@ -530,6 +530,9 @@ func (a *ActivityLog) availableLogs(ctx context.Context) ([]time.Time, error) {
 		if err != nil {
 			return nil, err
 		}
+		if time.After(upTo) {
```

Comment:
It always trips me up when variable names shadow package names. Would you be open to changing this variable name?

Reply:
done!